ca/config.go

```diff
@@ -135,6 +135,13 @@ func (s *SecurityConfig) UpdateRootCA(cert, key []byte, certExpiry time.Duration
 	return err
 }
 
+// loadNewRootCA replaces the root CA with a new root CA
+func (s *SecurityConfig) loadNewRootCA(rootCA *RootCA) {
+	s.mu.Lock()
+	defer s.mu.Unlock()
+	s.rootCA = rootCA
+}
```

Comment:
> I don't think non-leader renew their own certs. IIRC all of them are calling the manager for rotation, even though they are technically capable of doing it on their own.

I think in theory that's true, because if the `RootCA` object in the security config is not a signer, it requests TLS certificates from a remote.  However, if a node is a leader, it's `RootCA` object is a signer.  If it then gets demoted, I don't think it then throws out the signing part, so I think then technically it could issue its own.

I don't think there's an issue with allowing all managers to issue their own TLS certs for themselves.  In fact, if we want the root certificate updates to be propagated via raft, we probably do want managers to issue their own TLS certs?  It would make the logic simpler (load the root cert and key if I'm the leader - otherwise, load just the cert).

I'm also not sure that the signer part is removed from the `RootCA` if a manager that was a leader is demoted (which it should be).  I'll make sure to test that.